spec/spec_helper.rb

```diff
@@ -48,7 +48,7 @@ module BlockScore
     Kernel.srand config.seed
 
     config.around do |example|
-      Timeout.timeout(10, &example)
+      Timeout.timeout(6, &example)
```

Comment:
In our conversation the other day we talked about pushing it down to 3 seconds. It is excessive at 6 seconds IMO.